fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1608,6 +1609,14 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         }
     }
 
+    /**
+     *  Times that the cookie banner re-engagement dialog has been shown.
+     */
+    internal var cookieBannerReEngagementDialogShowsCount = counterPreference(
```

Comment:
❓ can it be a val as we're not changing this object anywhere else?

Reply:
Good catch updating it, thanks!